### ISSUE TITLE
When editing a entity in the diagram perspective the form looses focus to quick

### ISSUE THREAD

**Issue by willem-xbreeze** [CrossBreezeNL/crossmodel#84]
See attached video:
- In diagram: Select entity, 
- Add attribute
- Select the column name with mouse (since i want to overwrite it)
- If I move the mouse byond the form boundary the form disappears.

Also: When adding attribute the focus goes to the new column name field, that is good, but the default text should be selected by default instead of placing the cursor behind it.

https://github.com/user-attachments/assets/34ad132a-2fcc-4309-9186-56c1c9854ab9

**Comment by harmen-xb** [comment]
@willem-xbreeze 

I think it's because you drag from the form onto the diagram and then let go of the mouse in the diagram area. Which is a mouse action there.

What would be the desired behavior here? Do you want to ask to save? Or not switch to a different form in the property window as long as you don't right-click on an object in the diagram?

**Comment by willem-xbreeze** [comment]
@harmen-xb  correct, when you mousedown in the form, and mouseup on the diagram the entity gets deselected. I think this should not happen unless you click/mousedown in te diagram.
so on mouseup in the diagram, nothing should happen if the mousedown before mouseup was in the form. Or at least the selected entity should not change.
